src/wallet/api/wallet.cpp

```diff
@@ -1562,6 +1574,30 @@ size_t WalletImpl::importMultisigImages(const vector<string>& images) {
     return 0;
 }
 
+size_t WalletImpl::importMultisigImages(const std::string& input, bool isFile) {
+    try {
+        clearStatus();
+        checkMultisigWalletReady(m_wallet);
+
+        string data;
+        std::vector<cryptonote::blobdata> images;
+        if (isFile) {
+            if (!m_wallet->load_from_file(input, data)) {
```

Comment:
This also seems like a case where you have two functions living in the body of one function. If you want to share the pre and post code, there are other options than a single function, particularly considering this same pre/post-amble exists in pretty much every multi-sig function, so your sharing should probably either be wider or narrower.